### PR TITLE
Feature/mc 13917 docs for retrieving and updating origin settings for a site

### PR DESCRIPTION
**Pull request by BajajSid**
#### Changes made
Added docs for retrieving and updating origin settings of a site

#### Related PRs
- [https://github.com/cloudops/cloudmc-stackpath-plugin/pull/166, https://github.com/cloudops/cloudmc-stackpath-plugin/pull/189]

### CODE REVIEW

File: source/includes/stackpath/_origin_settings.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+```json
+{
+    "siteId": "eb3ecdbe-d73b-40e6-a263-166accba75ed",
+    "stackId": "144c3d16-2879-4286-82b6-75e58a6ec1cb",
+    "scopeConfigurationId": "45ab2429-ca8d-4145-acc3-f6aea6e8dbef",
+	"pullProtocol": "MATCH",
+	"hostHeader": "marvel.com",
+	"origin": {
+        "id": "d92531f4-28f5-456d-9b5f-d1cbb17ccfee"
```
Comment (patrickspensieri) [inline source/includes/stackpath/_origin_settings.md]: Minor formatting issue here

File: source/includes/stackpath/_origin_settings.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+`backupOrigin.username` <br/> *string* | Username to use when authenticating with the backup origin. 
+`backupOrigin.password` <br/> *string* | Password to use when authenticating with the backup origin. 
+`backupOrigin.sslValidationEnabled` <br/> *boolean* | Specifies if SSL validation for the backup origin is enabled. 
+`backupOrigin.commonCertificateName` <br/> *string* | Common name to validate SSL origin requests against.
+`backupOrigin.path` <br/> *string* | The backup origin's HTTP request path.
+`backupOrigin.hostname` <br/> *string* | The backup origin's HTTP request hostname. Can be a valid IPv4 address or a valid domain name. 
+`backupOrigin.port` <br/> *int* | The backup origin's HTTP port. Must be one of [80, 8080, 443, 1935, 9091].
+`backupOrigin.securePort` <br/> *int* | The backup origin's HTTPS port. Must be one of [80, 8080, 443, 1935, 9091].
+`backupOrigin.priority` <br/> *string* | The backup origin's priority to the scope. Must be one of ["ZERO", "ONE"]. "ZERO" has the highest priority.
```
Comment (patrickspensieri) [inline source/includes/stackpath/_origin_settings.md]: Does setting `backupOrigin.priority` to `ONE` replace the `origin`? If so, maybe we can be more explicit about that.

Comment (BajajSid) [reply source/includes/stackpath/_origin_settings.md]: 9199096

File: source/includes/stackpath/_origin_settings.md (hunk 1 above)
Comment (BajajSid) [reply source/includes/stackpath/_origin_settings.md]: 9199096
e0028ff

**Review by patrickspensieri — APPROVED** [review]
Thanks for the prompt changes

**Review by evanluc — APPROVED** [review]

**Pull request merged**